README.md

```diff
@@ -54,15 +54,18 @@ The following sections document how to download, install, configure, use and tro
   - [Downloading the files](#downloading-the-files)
   - [Extracting the files](#extracting-the-files)
   - [Verifying integrity of the download](#verifying-integrity-of-the-download)
-- [Configuration](#configuration)
-  - [Verifying YAML correctness](#verifying-yaml-correctness)
-- [Example INFORM jobs](#example-inform-jobs)
+- [Building from source](doc/build_from_source.md)
 - [Installation](#installation)
   - [Checking MotoPlus configuration](#checking-motoplus-configuration)
-  - [DX200, YRC1000 and YRC1000micro](#dx200-yrc1000-and-yrc1000micro)
-- [Building from source](doc/build_from_source.md)
-- [Updating the configuration](#updating-the-configuration)
-- [The micro-ROS Agent](#the-micro-ros-agent)
+  - [Installing MotoROS2 out file](#installing-motoros2-out-file)
+- [Example INFORM jobs](#example-inform-jobs)
+- [Understanding the micro-ROS Agent](#understanding-the-micro-ros-agent)
+- [Configuration](#configuration)
+  - [Network Configuration](#network-configuration)
```

Comment:
```suggestion
  - [Network Configuration](doc/network_configuration.md)
```

Better to skip straight to that document rather than forcing another click.